src/topology/homotopy/casts.lean

```diff
@@ -0,0 +1,78 @@
+/-
+Copyright (c) 2022 Praneeth Kolichala. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Praneeth Kolichala
+-/
+import topology.homotopy.fundamental_groupoid
+
+
+/-!
+# Fundamental group casts
```

Comment:
I don't see the point of this file. It feels like you're reinventing the wheel.

Reply:
If you have any suggestions, I'd appreciate it. The reason we want this abbreviation is because otherwise, when using `eq_to_hom`, we'd need to write out the type explicitly every time, since it's hard to get Lean to translate `x=y` in the topological space to `x=y` in the fundamental groupoid.

Reply:
I think I've simplified this a bit. I used `heq` and `conj_eq_to_hom_iff_heq` which was recently added to mathlib. Hopefully this is sufficient.